portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase

```diff
@@ -30,12 +30,39 @@ definition {
 	}
 
 	@description = "Confirm that the user can change the pagination on the dataset admin page"
-	@ignore = "Test Stub"
 	@priority = 4
 	test CanChangePaginationInAdminPage {
+		task ("Given create 6 data sets") {
+			for (var listDataSets : list "1,2,3,4,5,6") {
+				LexiconEntry.gotoAdd();
+
+				PortletEntry.inputName(name = "Test Dataset");
+
+				FrontendDataSetAdmin.selectProvider(key_type = "Channel");
+
+				PortletEntry.save();
```

Comment:
Hey @alexmds2004 Could you create a macro for create a dataset and call it 6 times?

Reply:
Yes